Earthfile

```diff
@@ -55,6 +55,14 @@ install-rust:
     RUN cargo --version
     RUN rustc --version
 
+formatting-check:
+    FROM +install-rust
+    COPY Cargo.toml Cargo.toml
+    COPY rustfmt.toml rustfmt.toml
+    COPY crates/ crates/
+    COPY sql-to-dbsp-compiler/lib sql-to-dbsp-compiler/lib
```

Comment:
We also need to check the java code at some point. I suggest using checkstyle but it's not going to be pretty when we first run it.

Reply:
If process slows you down without a clear benefit the process is wrong

Reply:
enforcing consistent formatting across a team of people is a clear benefit to me?